packages/app/src/server/service/comment.ts

```diff
@@ -7,6 +7,9 @@ import { stringifySnapshot } from '~/models/serializers/in-app-notification-snap
 import loggerFactory from '../../utils/logger';
 import Crowi from '../crowi';
 
+// TODO import and implement UserNotificationService
+// import UserNotificationService from './user-notification';
```

Comment:
UserNotificationService has no relation to this feature.
Please investigate `service/slack-integration.ts` and `packages/slackbot-proxy` at first.